### DISCUSSION TITLE
When to release next stable version ?

### DISCUSSION THREAD

**Discussion by shelllet** [microsoft/WindowsAppSDK#1794] [Feedback wanted]
the version `1.0.0-preview3` ,`co_await` does't return to ui thread. [co_await does't return to ui thread. when the call PickSingleFolderAsync is completed](https://github.com/microsoft/WindowsAppSDK/issues/1782).

this afffects a lot of code, when to release next stable version ?

my app was blocked by these bugs for a long time!

1.0.0-p1 and 1.0.0-p2 are all deprecated. there is no available version for us.

![image](https://user-images.githubusercontent.com/16729017/141737414-e7fc1d32-aa31-4e8f-a695-ab25c898c808.png)

**Comment by andrewleader** [comment]
Hi @shelllet, we'll help you with the `co_await` issue on the thread you created.

As for when 1.0 Stable is releasing, hopefully this week!

**Reply by DrusTheAxe** [reply]
This week indeed :-)

[1.0 Stable Release is LIVE! #1806](https://github.com/microsoft/WindowsAppSDK/discussions/1806)

**Reply by shelllet** [reply]
Great!,  `x:uid` don't work in unpacakged app, `co_await` don't work in packaged app! what should i do in 1.0 stable?
